linkerd/app/inbound/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,23 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use libfuzzer_sys::arbitrary::Arbitrary;
+extern crate hyper;
+
+#[derive(Debug, Arbitrary)]
+struct TransportHeaderSpec {
+        uri: Vec<u8>,
+        header1: Vec<u8>,
+        header2: Vec<u8>,
```

Comment:
This naming suggests to me that this is two separate headers, but actually it's a header name and a header value. Can we change the naming to reflect this?